test/e2e/cypress/e2e/hosts_overview.cy.js

```diff
@@ -173,8 +173,10 @@ context('Hosts Overview', () => {
 
   describe('Deregistration', () => {
     const hostToDeregister = {
-      name: 'vmdrbddev01',
-      id: '240f96b1-8d26-53b7-9e99-ffb0f2e735bf',
+      name: 'vmhdbdev01',
+      id: '13e8c25c-3180-5a9a-95c8-51ec38e50cfc',
+      tablePos: 7,
```

Comment:
This looks so ugly :sweat_smile: 
I would rename it to `tableRow` anyway

Reply:
I improved it a bit by removing the dependency on the table position, should look better